gui/default/syncthing/core/syncthingController.js

```diff
@@ -810,7 +811,7 @@ angular.module('syncthing.core')
         };
 
         $scope.deviceName = function (deviceCfg) {
-            if (typeof deviceCfg === 'undefined') {
+            if (typeof deviceCfg === 'undefined' || deviceCfg.deviceID === undefined) {
```

Comment:
Sorry for that ... ROFL, just wondered some two minutes what your issue is. :dancers: